tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1490,7 +1490,7 @@ func visitProductPage(timeout time.Duration, wantStatus int, headers ...*header)
 
 // visitWithApp visits the given url by curl in the given container.
 func visitWithApp(url string, pod string, container string, num int) error {
-	cmd := fmt.Sprintf("kubectl exec %s -n %s -c %s -- sh -c 'i=1; while [[ $i -le %d ]]; do curl -m 0.1 -i -s %s; let i=i+1; done'",
+	cmd := fmt.Sprintf("kubectl exec %s -n %s -c %s -- sh -c 'i=1; while [[ $i -le %d ]]; do curl -m 0.5 -i -s %s; let i=i+1; done'",
```

Comment:
is -m necessary?

Reply:
I think it is generally good to have some timeout just in case request taking forever to finish?

Reply:
if there is a chance of request taking forever, yes.

Otherwise, it will always remain a source of flakiness considering the distribution of the request time.

Or you may bump it up more to rule out hanging but still permit the longer than usual requests?

Reply:
Done.